ceph-ansible-pr-syntax-check/build/build

```diff
@@ -82,6 +82,46 @@ function test_ceph_release_in_ceph_default {
   echo "No '- ceph_release_num[ceph_release]' statements found in ceph-defaults role!" && return 0
 }
 
+function check_if_a_keyword_is_used() {
+  searchstring=$1
+
+  if [ "$2" -lt 1 ]; then
+    exit -2
+  elif [ "$2" -eq 1 ]; then
+    pathspec="HEAD"
+  else
+    pathspec="HEAD~"$(($2 - 1))
+  fi
+
+  count=0
+  output=$(git grep -c "$searchstring" $pathspec | awk -F ":" '{print $3}')
```

Comment:
`git grep` would search the entire codebase and not just the lines added/modified.